### ISSUE TITLE
Price Checker script not working

### ISSUE THREAD

**Issue by yourfac377** [w35l3y/userscripts#73]
**Browser:** Chrome<br />
**Plugin:** Tampermonkey <br />
**Script:** Price Checker<br />
**Accessed url:** https://www.neopets.com/space/coincidence.phtml<br />
**Installation url:** <br />
**Error Message:** No error message was displayed<br />
**Obs.:** Script seems to be running as it should it seems, but no prices are being displayed <br />
**Expected behavior:** Update script to new accommodate new security protocols, PRETTY PLEASE =) <br />
**Actual behavior:** script acts as though it is checking prices, no prices are displayed <br />
**Steps to reproduce:**<br />
 1. - go to any page script runs on
 2. - attempt to run script

**Comment by mediterra2** [comment]
yep, the 2 most important scripts do not work now (battledome & price check).  Quick price check also doesn't work, but others seem ok I think.

**Comment by w35l3y** [comment]
I am not having the spare time needed to keep some scripts.
Pull requests are always welcome.

**Comment by yourfac377** [comment]
I'm not sure what a pull request is. I barely understood how to file that the script isn't working and solely made an account for that purpose. I understand that you're unable to fix it. 
Thank you for getting back.
 I rely so heavily on this script that I don't know if I'll be able to continue playing without it. Maybe I'll just have to fork out the cash for super shop wizard

**Comment by mediterra2** [comment]
yourfac377, I don't think there is currently a replacement for it on clraik.com, but if you make a request, there's a good chance that Bat will make one.  He's pretty good at churning out scripts.  There's also a battledome script on Clraik that works reasonably well but not for weapons that disappear after their use.

**Comment by yourfac377** [comment]
I tried putting in a request there but it seems like they don't trust new users and I don't have the time or credit to get them to stop taking around me and try making it for me.

**Comment by mediterra2** [comment]
I haven't paid for VIP for a while and I notice they don't even allow comments on a lot of the threads related to programs/scripts anymore.  I will prolly buy one soon anyway because I use several of Bat's scripts that may need updating.

Anyway, I bet this $ script would be a pretty simple fix.  It's probably just a formality like using http* instead of http in some hardcoding (or something similarly basic).  The script still shows the $ but when you click on it, no results are returned.  I don't know what a pull request is either, btw, but maybe Bat would be willing to do one.

**Comment by Septimustache** [comment]
I've tried replacing the http with https or http* anywhere that references neopets.com in the source code, but it doesn't restore full functionality. If Bat ends up making a new script with similar functionality, please update here.

**Comment by yourfac377** [comment]
Has anyone fixed this script or written a new one yet? or should i just give up on ever having this one again? I miss it so much =(

**Comment by mlnl221** [comment]
I have a fork, but I havent been able to figure out how to fix the price checker. I got it working for a few pages but none for quests. 
Here is my Fork of the repo (install the scripts in this repo to get it working)
https://github.com/minicoz/NeopetsUserscripts/tree/master

**Comment by mlnl221** [comment]
Actually i just fixed it, I got it working for the new layout again.
